src/main/java/com/budgetControlGroup/budgetControl/controllers/BudgetCreateController.java

```diff
@@ -1,13 +1,14 @@
 package com.budgetControlGroup.budgetControl.controllers;
 
 import com.budgetControlGroup.budgetControl.Models.Budget;
-import com.budgetControlGroup.budgetControl.Models.User;
+import com.budgetControlGroup.budgetControl.Models.Category_List;
 import com.budgetControlGroup.budgetControl.workflows.BudgetCreateWorkflow;
-import com.budgetControlGroup.budgetControl.workflows.LoginWorkflow;
-import com.budgetControlGroup.budgetControl.workflows.RegisterWorkflow;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
 @RequestMapping("/budgetCreateController")
```

Comment:
This is the line I'm talking about. You may also be able to remove it from line 24